src/Codeception/Module/Symfony/DoctrineAssertionsTrait.php

```diff
@@ -43,65 +41,42 @@ public function grabNumRecords(string $entityClass, array $criteria = []): int
     }
 
     /**
-     * Grab a Doctrine entity repository.
-     * Works with objects, entities, repositories, and repository interfaces.
+     * Obtains the Doctrine entity repository {@see EntityRepository}
+     * for a given entity, repository class or interface.
      *
      * ```php
      * <?php
-     * $I->grabRepository($user);
-     * $I->grabRepository(User::class);
-     * $I->grabRepository(UserRepository::class);
-     * $I->grabRepository(UserRepositoryInterface::class);
+     * $I->grabRepository($user);                          // entity object
+     * $I->grabRepository(User::class);                    // entity class
+     * $I->grabRepository(UserRepository::class);          // concrete repo
+     * $I->grabRepository(UserRepositoryInterface::class); // interface
      * ```
+     *
+     * @param  object|class-string $mixed
+     * @return EntityRepository<object>
```

Comment:
```suggestion
     * @phpstan-template T of object
     * @phpstan-param  T|class-string<T> $mixed
     * @phpstan-return EntityRepository<T>
```

Use generics should make the type hints even more precise. WDYT?